src/components/__fixtures__/helpers.js

```diff
@@ -1,7 +1,10 @@
-import { mount } from '@vue/test-utils'
 import FontAwesomeIcon from '../FontAwesomeIcon'
+import pkg from '@fortawesome/fontawesome-svg-core/package.json'
+import { mount } from '@vue/test-utils'
 import { parse } from '@fortawesome/fontawesome-svg-core'
 
+const coreMajorVersion = Number.parseInt(pkg.version?.split('.')[0], 10) || 0
```

Comment:
@robmadole ---
Do you have any hesitations using this here for our tests  (importing `pkg`)?

I could not think of a reason not to use here... and it is working out really well for testing certain v7 small batch icon family styles.